job_executor/adapter/local_storage.py

```diff
@@ -423,3 +423,13 @@ def delete_archived_input(dataset_name: str):
     archived_file: Path = INPUT_DIR / f"archive/{dataset_name}.tar"
     if archived_file.is_file():
         os.remove(archived_file)
+
+
+def get_size_in_bytes(dataset_name: str) -> int:
+    """
```

Comment:
Rename suggestion: get_input_tar_size_in_bytes
Or something else a bit more explicit about the state of the data (or something in the docstring is fine too)